integration-tests/bootstrap/bootstrap_test.go

```diff
@@ -0,0 +1,24 @@
+package bootstrap
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openebs/maya/integration-tests/kubeconnect"
+)
+
+var _ = Describe("Connect", func() {
+	Describe("to kubernetes", func() {
```

Comment:
Avoid nested functions when not required

Reply:
done